OpenRA.Mods.Common/Util.cs

```diff
@@ -212,19 +212,19 @@ public static int RandomDelay(World world, int[] range)
 		public static string FriendlyTypeName(Type t)
 		{
 			if (t.IsGenericType && t.GetGenericTypeDefinition() == typeof(HashSet<>))
-				return "Set of {0}".F(t.GetGenericArguments().Select(FriendlyTypeName).ToArray());
+				return string.Format("Set of {0}", t.GetGenericArguments().Select(FriendlyTypeName).ToArray());
 
 			if (t.IsGenericType && t.GetGenericTypeDefinition() == typeof(Dictionary<,>))
-				return "Mapping of {0} to {1}".F(t.GetGenericArguments().Select(FriendlyTypeName).ToArray());
+				return string.Format("Mapping of {0} to {1}", t.GetGenericArguments().Select(FriendlyTypeName).ToArray());
```

Comment:
Bogus format string broke your script?

Reply:
Looks like it, do we want a string with the key and value?

Reply:
I think so yeah.